packages/nextra/src/types.ts

```diff
@@ -2,15 +2,12 @@ import { Heading as MDASTHeading } from 'mdast'
 import { ProcessorOptions } from '@mdx-js/mdx'
 import { PageMapCache } from './plugin'
 
-export interface LoaderOptions {
-  theme: Theme
-  themeConfig: string
+export interface LoaderOptions extends NextraConfig {
   locales: string[]
   defaultLocale: string
-  unstable_staticImage: boolean
-  unstable_flexsearch: boolean
```

Comment:
it's better to extends `NextraConfig` since we merge two objects in `src/index.js` file to avoid copy-paste the same